src/ImageSharp/Formats/Jpeg/Components/Encoder/YCbCrForwardConverter{TPixel}.cs

```diff
@@ -48,7 +47,12 @@ namespace SixLabors.ImageSharp.Formats.Jpeg.Components.Encoder
         public static YCbCrForwardConverter<TPixel> Create()
         {
             var result = default(YCbCrForwardConverter<TPixel>);
-            result.colorTables = RgbToYCbCrTables.Create();
+            if (RgbToYCbCrConverterVectorized.IsSupported)
```

Comment:
shouldn't this be `if (!RgbToYCbCrConverterVectorized.IsSupported)` instead?

Reply:
Just fixed this a second ago, well spotted 👍